napari/components/viewer_model.py

```diff
@@ -965,8 +965,8 @@ def open_sample(
         sample : str
             name of the sample
         reader_plugin : str, optional
-            reader plugin to pass to viewer.open (only used if the sample data
-            is a string).  by default None.
+            reader plugin to use, passed to ``viewer.open`` (only used if the
+            sample data is a URI).  By default None.
```

Comment:
Probably a good idea to put the full definition of the URI acronym here too.

Reply:
Thanks, done!